modules/tinymce/src/plugins/visualchars/test/ts/browser/PluginTest.ts

```diff
@@ -1,51 +1,53 @@
-import { ApproxStructure, Assertions, Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
+// import { ApproxStructure, Assertions, Log, Pipeline } from '@ephox/agar';
```

Comment:
Do we still need this? Also reminder for that lint rule discussion from this morning.

Reply:
Sorry I missed that